### ISSUE TITLE
"Don't take foreground window on Window servers"

### ISSUE THREAD

**Issue by ghost** [input-leap/input-leap#47]
> This issue has been migrated from old Barrier Github repository https://github.com/debauchee/barrier/issues/47
> 
> Issue created on: 2018-05-23 by @johnny-mac
> Issue last updated on: 2018-06-30

### Operating Systems ###
Artix Linux

Server: Artix & Void

Client: Artix & Devuan

### Barrier Version ###

2.1.1 

### Steps to reproduce bug ###

"Don't take foreground window on Window servers"

I'm sorry if I misunderstand this but shouldn't this option stop barrier from making my game window become 'not the main window' when I move my mouse to the other screen? Its really annoying, I'd like to use this while gaming & watching videos but this messes my game up & sometimes my PS3 controller will stop working.... I take my mouse off my main PC to my secondary one & it makes it not the main window, the active window... but its not like I clicked on anything else on that window, I just moved my mouse to the other one... Apparently this works for some people but I can't seem to make it work for me.

---

> Commented on: 2018-06-01 by @johnny-mac

The "Dead Corners" don't work either, I enabled dead corners for top left & top right corners but the corners most certainly are not dead, but that's not anywhere near as important as the original issue above, but maybe it has something to do with it?
When I disable clipboard sharing it appears clipboard sharing still works too. 

Don't take foreground window on Windows servers... oh... Windows, like microsoft windows... is that what this means? :/ why isn't there an option like that for Linux? I don't want the window to become inactive when I put my mouse on the other screen, I want to keep my mouse & keyboard on the computer I use to play videos while I'm gaming on my main PC with a PS3 controller... Why isn't this possible? Surely I am not the only one to suggest such a thing? 

---

> Commented on: 2018-06-30 by @walker0643

Not quite, @johnny-mac. Your mouse never really leaves your primary screen in the technical sense... the same way a magician's fingers never leave his hands... part of the magic of Barrier is that it simulates a single mouse being connected to and communicating with multiple machines while only being physically connected to one of them.

TLDR: When your mouse is active on another screen it can't also be active inside your game window on the primary screen. Same with the keyboard.

---

> Closed on: 2018-06-30 by @walker0643
